### PR TITLE
first step for tags support - issue #25

### PR DESCRIPTION
**Pull request by pierky**
Needs tags support in Cousteau https://github.com/RIPE-NCC/ripe-atlas-cousteau/pull/5

### CODE REVIEW

**Comment by astrikos** [comment]
Let's wait until we release new cousteau version before merging that. It should be harmless but better wait. I am planning to do so next week, which will support python3 as well :) Until then make yourself assignee for the issue so we know you work on this one.

File: ripe/atlas/tools/commands/measure.py [hunk 1]
```diff
@@ -480,6 +496,20 @@ def _get_source_kwargs(self):
...
             r["value"] = self.arguments.from_measurement
 
+        if self.arguments.include_tag or self.arguments.exclude_tag:
+            r["tags"] = {}
+            r["tags"]["include"] = self.arguments.include_tag or []
+            r["tags"]["exclude"] = self.arguments.exclude_tag or []
+        else:
+            smart_tags = conf["smart-tags"]
+            try:
+                r["tags"] = smart_tags[self.arguments.type][str(self.arguments.af)]
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/measure.py]: I'm having trouble understanding what's going on here.  Is the intention to have `smart_tags` contain keys of `af`, as well as `ping`, `traceroute`, etc.?  What's the benefit here?

Comment (pierky) [inline ripe/atlas/tools/commands/measure.py]: Just a shorter way to say `conf["smart-tags"]`.
More comments on my idea on Issue https://github.com/RIPE-NCC/ripe-atlas-tools/issues/25#issuecomment-148123848

**Comment by danielquinn** [comment]
Closing this as it's superceded by #49

**Pull request closed without merging**